setup/ebill_paynet_customer_free_ref/odoo/addons/ebill_paynet_customer_free_ref

```diff
@@ -0,0 +1 @@
+../../../../ebill_paynet_customer_free_ref
```

Comment:
Aren't the contents of ```setup``` generated automatically by the tool ```black```? (same for the next file, *setup.py*)

Reply:
if you are adding a new module it's always better to include this in the PR because it makes possible to install the module via pip directly from the PR